book/en/src/internals/targets.md

```diff
@@ -0,0 +1,59 @@
+# Target Architecture
+
+While RTIC can currently target all Cortex-m devices there are some key architecure differences that 
+users should be aware. Namely the absence of hardware priority ceiling (BASEPRI) support in the 
+ARMv6-M and ARMv8-M-base architectures requires a few tweaks from RTIC to deliver the same 
+features. These differences result in two flavors of critical sections: priority ceiling, and source 
+masking. Table 1 below shows a list of Cortex-m processors and which type of critical section they 
+employ.
+
+#### *Table 1: Critical Section Implementation by Processor Architecture*
+
+| Processor  | Architecture | Priority Ceiling | Source Masking |
+| :--------- | :----------: | :--------------: | :------------: |
+| Cortex-M0  | ARMv6-M      |                  |     &#2713     |
+| Cortex-M0+ | ARMv6-M      |                  |     &#2713     |
+| Cortex-M3  | ARMv7-M      |      &#2713      |                |
+| Cortex-M4  | ARMv7-M      |      &#2713      |                |
+| Cortex-M23 | ARMv8-M-base |                  |     &#2713     |
+| Cortex-M33 | ARMv8-M-main |      &#2713      |                |
+| Cortex-M7  | ARMv7-M      |      &#2713      |                |
+
+## Priority Ceiling
+
+This implementation is covered in depth by Chapter 4.5 of this book.
```

Comment:
This is as you know currently not available on the rtic.rs book, only through the sources.

Additionally, numerical hardcoded references are tricky so please prefer using links to the actual target :)

Reply:
I wasn't sure how to capture the reference since it's not published. Should it be a URL to the github markdown source? Or propose the change, thanks :)